docs/ecosystem-user-event.mdx

```diff
@@ -49,7 +49,7 @@ in
 
 ### `click(element, eventInit, options)`
 
-Clicks `element`, depending on what `element` is it can have different side
+Clicks `element`, depending on what `element` it is, it can have different side
```

Comment:
```suggestion
Clicks `element`, depending on what `element` is clicked, calling `click()` can have different side
```

Really hard to follow what "it" is referring to

Reply:
Agreed 👍🏻